packages/components/src/components/Navigation/components/NavigationGroup/NavigationGroup.module.scss

```
@@ -0,0 +1,9 @@
+.navigationGroup {
+  display: flex;
+  flex-direction: column;
+  row-gap: var(--navigation--item-to-item-spacing);
+
+  &:not(:last-child) {
+    margin-bottom: var(--navigation--group-to-group-spacing);
+  }
```

```suggestion
  & + & {
    margin-top: var(--navigation--group-to-group-spacing);
  }
```

Finde dieser Selector drückt besser "zwischen zwei Groups" aus. Außerdem kann ja vielleicht in einer Navigation nach einer Group auch noch "etwas anderes" kommen, wie z.B. ein Avatar?

---

Hat sich erledigt, da die Groups noch durch Headings getrennt sind. Daher passt es so.